src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json

```diff
@@ -1,3 +1,44 @@
 {
-    "useWebAssemblyDebugging": true
-}
\ No newline at end of file
+  "iisSettings": {
+    //#if (WindowsAuth)
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+    //#else
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    //#endif
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8080",
+      //#if(RequiresHttps)
+      "sslPort": 44300
+      //#else
+      "sslPort": 0
+      //#endif
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "useWebAssemblyDebugging": true,
```

Comment:
This line `"useWebAssemblyDebugging": true,` should be removed. We no longer use it because `inspectUri` triggers the wasm debugging behavior on its own.